examples/fasthtml-response-streaming/app/main.py

```diff
@@ -0,0 +1,62 @@
+from anthropic import AnthropicBedrock
```

Comment:
Any reason to use AnthropicBedrock instead of boto3? If you need to use it, the README and requirementes.txt should be updated .

Reply:
Have made the correction, I prefer to use it because of the simple streaming support and making it easier to use the [Instructor](https://python.useinstructor.com/) module for structured output. The trade off is naturally it restricts the models to Anthropic only.